### PR TITLE
Measure kogito-editors-js coverage

### PR DESCRIPTION
**Pull request by jomarko**

### CODE REVIEW

File: kogito-editors-js/pom.xml [hunk 1]
```diff
@@ -21,9 +21,7 @@
     <sonar.test.inclusions>**/*test.ts,**/*test.tsx</sonar.test.inclusions>
     <sonar.javascript.lcov.reportPaths>
       packages/boxed-expression-component/coverage/lcov.info,
-      packages/dmn-loader/coverage/lcov.info,
       packages/feel-input-component/coverage/lcov.info,
-      packages/hello-world-component/coverage/lcov.info,
```
Comment (yesamer) [inline kogito-editors-js/pom.xml:26]: @jomarko Can you please clarify these 2 removals?

**Review by yesamer — APPROVED** [review]

File: kogito-editors-js/pom.xml (hunk 1 above)
Comment (romartin) [reply kogito-editors-js/pom.xml:26]: I am not really up to date on this guys, so I agree with Yeser about iinfo that look a bit weird. Anyway, adding coverage looks good to me, if this PR does not actually prevents from building acutal repos. :+1:

Comment (jomarko) [reply kogito-editors-js/pom.xml:26]: well, my intention was somehow optimize/minimize execution time. I know from global point of view (wating for whole CI) it is maybe even not noticeable, but I think we should try  to improve CI in such small steps as here, here are my assumptions, but I am open to your comments and suggestions:
- **dmn-loader** has no tests, so from my point of view doesn't make sense to try measure the coverage there
- **hello-world-component** has no tests and furthermore it is just a showcase not present in our product at all

**Comment by jomarko** [comment]
@yesamer @romartin hello could we please merge this PR?

**Comment by jomarko** [comment]
Coverage info available now
https://sonarcloud.io/code?id=org.kie.kogito%3Akogito-editors-java&selected=org.kie.kogito%3Akogito-editors-java%3Akogito-editors-js%2Fpackages

Screenshot from 2021-11-22 09-55-53

**Pull request merged**